eng/pipelines/templates/jobs/release.yml

```diff
@@ -83,10 +88,63 @@ jobs:
             Registry: https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-js/npm/registry/
             Tag: latest
 
+
+- job: ExtractVsixVersion
+  displayName: "Extract VSIX version from package.json"
+  steps:
+    - pwsh: |
+        $packageJson = Get-Content eng/vscode/package.json -Raw | ConvertFrom-Json
+        Write-Host "##vso[task.setvariable variable=VsixVersion;isOutput=true]$($packageJson.version)"
+      name: setVsixVersion
+      displayName: "Extract VSIX version from package.json"
+
+# Job: Release VSIX artifacts to Marketplace
+- job: ReleaseVSIX
```

Comment:
nit: should this be ReleaseVSCodeVSIX to make room for other marketplace VSIX pipelines?